mem.go

```
@@ -2,12 +2,61 @@ package sqrl
 
 import (
 	"fmt"
+	"math/big"
 
-	"github.com/shirou/gopsutil/mem"
+	"github.com/dmmcquay/likexian/host-stat-go"
+	"github.com/dmmcquay/sqrl/dustin/go-humanize"
 )
 
-func MemInfo() {
+func meminfo() (uint64, uint64) {
 	v, _ := mem.VirtualMemory()
+	var total uint64
+	total = v.Total
+	free := v.Free
+	return total, free
+}
+
+func swapinfo() (uint64, uint64) {
+	s, _ := mem.SwapMemory()
+	total := s.Total
+	free := s.Free
+	return total, free
+}
 
-	fmt.Println(v)
+func versioninfo() (string, string, string) {
+	i, _ := host_stat.GetHostInfo()
```

Same thing here.  Don't ignore errors! They can come back to haunt us....